### PR TITLE
Tell how to generate an SSH key

### PR DESCRIPTION
**Pull request by RickJWagner**
### What does this PR do?
Tells how to generate an SSH key.

### What issues does this PR fix or reference?
CRW-694

### Specify the version of the product this PR applies to. 
CRW 2.x

### CODE REVIEW

File: src/main/pages/che-7/end-user-guide/proc_accessing-a-git-repository-via-ssh.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ A common SSH key that works with all Git providers is present by default. To sta
 
 To generate an SSH key pair that only works with a particular Git provider:
 
-. Run the *SSH: generate key pair for particular host* command.
+. Run the *SSH: generate key pair for particular host* command. Do this by pressing 'F1' to bring up the dialogue box, then begin typing 'generate'.  You will be given options for SSH key generation.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_accessing-a-git-repository-via-ssh.adoc]:
```suggestion
. Run the *SSH: generate key pair for particular host* command. Do this by pressing kbd:[F1] to open the command dialogue box, then begin typing `generate`. You will be given options for SSH key generation.
```

**Review by rkratky — APPROVED** [review]
Thanks, @RickJWagner. A minor suggestion added.

**Comment by rkratky** [comment]
Also, @RickJWagner, please, include the "Signed-off-by footer" in your commit message, or the system won't allow merging (amend and force-push to modify the existing commit message).

**Comment by rkratky** [comment]
@RickJWagner, I'm seeing now there's been some attempts at this PR already. I don't want to make you jump through hoops, so I've submitted the change and the sign-off fix directly to your branch.

Please, let me know if you're happy with my change, and I'll be able to merge immediately.

**Comment by RickJWagner** [comment]
Hi @rkratky 
Thank you for your assistance!  
Yes, the alteration looks good.  And thank you for rescuing me from the 'hoops to be jumped through'.  :)

**Pull request merged**